spring-cloud-sleuth-samples/spring-cloud-sleuth-sample-test-core/src/main/java/tools/RequestSendingRunnable.java

```diff
@@ -50,9 +51,9 @@ public void run() {
 		log.info("Received the following response [{}]", responseEntity);
 	}
 
-	private RequestEntity requestWithTraceId(String traceId) {
+	private RequestEntity requestWithTraceId(Long traceId) {
 		HttpHeaders headers = new HttpHeaders();
-		headers.add(Trace.TRACE_ID_NAME, traceId);
+		headers.add(Trace.TRACE_ID_NAME, LongUtils.toString(traceId));
```

Comment:
These util classes :P When you introduce a type, you now have the problem of naming it. Util is a junk drawer, and toString doesn't really tell me anything. I have to look at call sites to see .. ahh this is for span ids! IOTW, this code is less about Longs and more about span ids, why not call it that?

`SpanId.toString(long)/fromString(String)` would be closer, as at least that indicates we are going in and out of the canonical string form of a span id.